com.unity.render-pipelines.high-definition/Editor/Material/MaterialHeaderScope.cs

```diff
@@ -26,6 +26,15 @@ internal struct MaterialHeaderScope : IDisposable
         int oldIndentLevel;
 #endif
 
+        /// <summary>
+        /// Creates a material header scope to display the foldout in the material UI.
+        /// </summary>
+        /// <param name="title">Title of the header.</param>
+        /// <param name="bitExpanded">Bit index to use for the.</param>
+        /// <param name="materialEditor">The current material editor.</param>
+        /// <param name="spaceAtEnd">Do the block shows a space at the bottom of it's UI.</param>
```

Comment:
Set this to true to make the block include space at the bottom of its UI. Set to false to not include any space.